emitter/text/emitter.go

```diff
@@ -0,0 +1,102 @@
+package text // import "github.com/vimeo/alog/emitter/text"
```

Comment:
Do we want a different package naming convention?

This one is `text`, is `json` a json log emitter?